packages/ui/src/components/textfield/textInput/TextInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { type ComponentPropsWithoutRef } from "react";
+import arrow from "../../../../assets/chevronDown.png";
+import { common, error, icon, input, textfieldWrapper } from "../style.css";
+type InputProps = ComponentPropsWithoutRef<"input"> & {
+  errorMessage?: string;
+  isSelect?: boolean;
+  show?: boolean;
+};
+
+const TextInput = ({
+  errorMessage,
+  disabled = false,
+  placeholder = "텍스트를 입력해주세요.",
+  isSelect = false,
+  show = false,
+  ...props
+}: InputProps) => {
+  let invalid = !!errorMessage;
+  return (
+    <div className={textfieldWrapper}>
+      <input
+        readOnly={isSelect}
+        disabled={disabled}
+        className={`${common({ disabled, invalid })}, ${input}`}
```

Comment:
className={`${common({ disabled, invalid })}, ${input}`}
여기서
class명 사이에 `,`가 있으면 안됨
```suggestion
        className={`${common({ disabled, invalid })} ${input}`}
```